File: algoexpert/python/medium/invertBinaryTree.py
def invertBinaryTree(root):
    queue = [root]

    while len(queue):
        for item in queue:
            node = queue.pop(0)

            temp = node.left
            node.left = node.right
            node.right = temp

            if node.left:
                queue.append(node.left)
            if node.right:
                queue.append(node.right)

    return root

File: algoexpert/javascript/medium/invertBinaryTree.js
function invertBinaryTree(root) {
  if (root === null) return null;
  const queue = [root];

  while (queue.length) {
    let len = queue.length;

    for (let i = 0; i < len; i++) {
      let node = queue.shift();

      temp = node.left;
      node.left = node.right;
      node.right = temp;

      if (node.left) queue.push(node.left);
      if (node.right) queue.push(node.right);
    }
  }
  return root;
};